src/command_modules/azure-cli-vm/tests/test_vm_commands.py

```diff
@@ -1677,7 +1678,7 @@ def body(self):
         self.cmd('vmss restart --resource-group {} --name {} --instance-ids *'.format(self.resource_group, self.ss_name))
         self._check_vms_power_state('PowerState/running', 'PowerState/starting')
         self.cmd('vmss stop --resource-group {} --name {} --instance-ids *'.format(self.resource_group, self.ss_name))
-        self._check_vms_power_state('PowerState/stopped')
+        self._check_vms_power_state('PowerState/stopped', 'PowerState/stopping')
```

Comment:
Well, since we already entries for other "seems like it shouldn't be but sometimes is" power states ("starting" in the lines above) I'm okay with merging this.